app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,11 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
-  contact: "",
-  tag: "",
+  role: null,
+  tags: [],
```

Comment:
Can you please point out why `tags` has been kept as empty array `[]` and not `null`, whereas `role` has been kept as `null`?

Reply:
Because  we can have multiple tags , hence we would go for an array & even if we remove the tags . We still have an empty array & not null.
For role, I have assumed one of the two roles ( Owner/ Employee) for right now.
Should i modify `role: [ ]`  in the code?

Reply:
>Should i modify role: [ ] in the code?

No need.